pay-api/src/pay_api/services/eft_service.py

```diff
@@ -60,6 +60,11 @@ def apply_credit(self, invoice: Invoice) -> None:
         self.create_receipt(invoice=invoice_model, payment=payment).save()
         self._release_payment(invoice=invoice)
 
+    def complete_post_invoice(self, invoice: Invoice, invoice_reference: InvoiceReference) -> None:
```

Comment:
Accidentally reverted this change in a previous PR, adding back in.

This allow payments to be properly released for EFT when invoices are created.